chromadb/__init__.py

```diff
@@ -1,10 +1,14 @@
 import chromadb.config
 import logging
+from chromadb.telemetry.events import ClientStartEvent
+from chromadb.telemetry.posthog import Posthog
 
 logger = logging.getLogger(__name__)
 
 __settings = chromadb.config.Settings()
 
+__version__ = "0.3.12"
```

Comment:
we could do this...

```
import chromadb

import subprocess

def get_package_version(package_name):
    output = subprocess.check_output(['pip', 'show', package_name]).decode('utf-8')
    for line in output.split('\n'):
        if line.startswith('Version:'):
            return line.split(' ')[1]

# Example usage:
version = get_package_version('chromadb')
print(version)
```

Reply:
We cannot rely on a users environment having pip installed.

From my research, having a __version__ flag is a common practice among python packages and has become something of an unofficial standard off of this PEP https://peps.python.org/pep-0396/

There are other programmatic approaches but they either require versions of python who's minimum is above our minimum or require new dependencies - which causes bloat.  See 
https://stackoverflow.com/questions/458550/standard-way-to-embed-version-into-python-package
https://stackoverflow.com/questions/20180543/how-do-i-check-the-versions-of-python-modules

I agree that this has some room for error, but as we mature our release process, I think we will have to be diligent our have tooling to allow us to not making simple mistakes on the version error.

Reply:
@HammadB this is a good point! I am not sure. If you can take a quick look at what other packages do, that would be great. Otherwise let's ship it.

Reply:
So seems like several python stdlib libraries just hardcode it
https://github.com/search?q=repo%3Apsf%2Frequests%20__version__&type=code

pytorch does the same
https://github.com/search?q=org%3Apytorch%20__version__&type=code

I think this is OK for now, I'll write up a release process and add it to the repo.

Reply:
@HammadB great! thanks!